commands/apps.go

```diff
@@ -169,3 +174,130 @@ func (c *AppsDeleteCmd) Run() (err error) {
 
 	return err
 }
+
+// AppsInitCmd creates and validates server.conf and package.json to prepare an app for deployment
+type AppsInitCmd struct {
+	StackName string `optional default:"nodejs-basic" short:"s" help:"Name of stack to deploy. Default is nodejs-basic"`
+	Force     bool   `optional short:"f" help:"Resets deployment specific files to their default configuration"`
+}
+
+func (c *AppsInitCmd) buildServerConf() []byte {
+	return []byte(
+		`location / {
+	proxy_set_header X-Forwarded-For $http_x_forwarded_for;
+	proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto;
+	proxy_set_header Host $host;
+	include /etc/nginx/section.module/node.conf;
+}
+
+location ~ "/next-proxy-hop/" {
+	proxy_set_header X-Forwarded-For $http_x_forwarded_for;
+	proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto;
+	proxy_set_header Host $host;
+	proxy_pass http://next-hop;
+}`)
+}
+
+// Run executes the command
+func (c *AppsInitCmd) Run() (err error) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	switch c.StackName {
+	case "nodejs-basic":
```

Comment:
is this necessary for now?

I imagine we might have other stacks if we ever do like... static site hosting or python/django as a module?

Also, maybe just like `-s node` or `--stack node` would be easier to write

Reply:
it's for future proofing, for when we offer more than node or hosting varnish. end users won't have to worry about this for now since the command defaults to 'nodejs-basic'